public/python/model_checker.py

```
@@ -68,6 +68,7 @@ def check_dependencies():
 check_dependencies()
 
 import ifcopenshell
+from ifcopenshell.util.schema import get_fallback_schema
```

Pylint (3.3.7)

[error] Unable to import 'ifcopenshell.util.schema'

(E0401)